### DISCUSSION TITLE
C++版本paddleOCR文字识别模型内存泄漏问题

### DISCUSSION THREAD

**Discussion by Lebron-Harden** [PaddlePaddle/PaddleOCR#13901] [Q&A] [answered]
<br>    
    如题，最近将paddleOCR部署到本机上进行测试（环境是Windows11），按照C++部署教程来的，C++预测库paddle_inference预测库下载的如下：

![image](https://github.com/user-attachments/assets/1e0f63cf-101a-4870-ab22-22406f3a4b61)

测试是只使用了文字识别模型，检测是用的别的模型。测试中发现，如果一直循环识别同一张图片，内存不会涨，但是识别一个目录下面的不同图片，内存会一直涨，虽然涨的不快，但明显是有内存泄漏的。用到的动态库如下：
![image](https://github.com/user-attachments/assets/573549da-3b1e-455d-b038-60ff5a4d15f7)

<br>  <br>  
我自己排查过，应该不是new模型指针然后释放不了的问题。之前在别的issue看到，说是什么tensor复用导致，这样也似乎能解释循环识别一张图片不会内存泄漏。那我这个情况应该怎么解决内存泄漏的问题呢？求大佬帮助

**Comment by GreatV** [comment]
paddle inference 换成 2.6 或者 3.0 beta 试试看

**Reply by Lebron-Harden** [reply]
请问paddle_inference2.6或者3.0beta版本在哪里获取呢？我这边在官方找的windows预测库，只有2.3.2版本的

**Reply by GreatV** [reply, marked as answer]
https://www.paddlepaddle.org.cn/inference/v3.0/guides/install/download_lib.html#windows

**Reply by Lebron-Harden** [reply]
感谢
